matter_server/server/device_controller.py

```diff
@@ -315,6 +315,14 @@ async def commission_with_code(
             LOGGER.info("Commissioned Node ID: %s vs %s", commissioned_node_id, node_id)
             if commissioned_node_id != node_id:
                 raise RuntimeError("Returned Node ID must match requested Node ID")
+
+            await self._chip_device_controller.send_command(
+                node_id,
+                0,
+                Clusters.OperationalCredentials.Commands.UpdateFabricLabel(
+                    "Home Assistant"
```

Comment:
I think we should provide this value when starting the server

Reply:
Apple sets the "Home Name" as label, so yes maybe something the user can configure in settings?

Reply:
In Home Assistant we have a instance name (aka. location_name, which is also used in DNS-SD as instance name). I've intended to use that:

![image](https://github.com/user-attachments/assets/7a807b8a-de04-41f4-bb07-a441a5fea95b)